dist/js/apps/events-page/featuredEventsWidgetApp.js

```diff
@@ -219,7 +219,7 @@
 		requestChunkSize: 10,
 		ageDisclaimer: {
 			message: 'Children under 8 must be accompanied by adult',
```

Comment:
'Children under 8 must be accompanied by **_an_** adult',

Reply:
will run this by dan